DataWorkerService.Tests/AutomationChecks/Matches/MatchGameCountCheckTests.cs

```diff
@@ -107,4 +107,45 @@ MatchRejectionReason expectedRejectionReason
         Assert.Equal(expectedPass, actualPass);
         Assert.Equal(expectedRejectionReason, match.RejectionReason);
     }
+
+    [Fact]
+    public void Check_Passes_WhenApplyingWarningFlag()
+    {
+        // Arrange
+        Match match = SeededMatch.Generate(warningFlags: MatchWarningFlags.None);
+
+        SeededGame.Generate(verificationStatus: VerificationStatus.Verified, match: match);
+        SeededGame.Generate(verificationStatus: VerificationStatus.Verified, match: match);
+        SeededGame.Generate(verificationStatus: VerificationStatus.Verified, match: match);
+        SeededGame.Generate(verificationStatus: VerificationStatus.Verified, match: match);
+
```

Comment:
for loop me thinks

Reply:
methinks it will take the same amount of space :P

Reply:
true wisdom lies not in the space it takes, but in the loops that spare us the tedium of repetition!

Reply:
me when I wrote the one line and then alt + shift + down arrow 3 times in a row